org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -30,6 +30,28 @@ Parameters:
     Type: String
     Default: '906769aa66-5d23a723-54f3-4c08-a67b-311e555f4e85'
 
+  imageCentralAmiLibrarianGroup:
+    Type: String
+    Default: '906769aa66-8bfbe918-2d30-467e-9e98-92232e9f8410'
```

Comment:
where does this string ('906769aa66-8bfbe918-2d30-467e-9e98-92232e9f8410') come from?

Reply:
it's the AWS SSO group ID which is generated by AWS.  You can find it on the AWS SSO configuration and it looks like this.. https://pasteboard.co/K1BlqLw.png

Reply:
I see:  Elsewhere when we have one CF script to create a resource we then pass the generated ID (for example, an ARN) automatically to the CF script(s) that use it.  Why don't we do that here rather than hard coding the ID into the script?

Reply:
Because we don't generate those group IDs in our templates so we don't have a reference to them.  That's why we need to hard code them into the template.  We use the [jumpcloud SCIM feature](https://jumpcloud.com/blog/scim-provisioning-defined) and that is the thing that triggers AWS to generate the IDs.